_posts/2016-01-25-composer-mailchimp-subscriptions.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: "How to Subscribe Users to MailChimp Lists in a Druapl Custom Module"
```

Comment:
I'm not sure we are 100% consistent but most of our titles are lowercased, so it would be "How to subscribe users to MailChimp lists in a Drupal custom module"